### ISSUE TITLE
Obtaining version information returns error code 1

### ISSUE THREAD

**Issue by andymccall** [envenomator/agon-ez80asm#97]
Hi,

I've noticed that obtaining the version information on ez80 returns the version information, but then continues actions, if you obtain the version information alone an error code is returned.  You may want to obtain the version information in pipelines to check and report on prerequisites are present.  As an error code is returned this isn't possible.

```
$ bin/ez80asm -v
ez80asm version 1.10, (C)2024 - Jeroen Venema
No input filename
Usage: ez80asm <filename> [output filename] [OPTION]

  -v    List version information
  -h    List help information
  -o    Org start address in hexadecimal format, default is 040000
  -b    Fillbyte in hexadecimal format, default is FF
  -a    ADL mode 1/0, default is 1
  -i    Ignore value truncation warnings
  -l    Listing to file with .lst extension
  -s    Export symbols
  -d    Direct listing to console
  -c    No color codes in output
  -x    Display assembly statistics

$ echo $?
1
```

If this is by design, then please let me know I'll get the help listing instead in my pipeline to check for the binary, but I suspect you should be able to get the version information and this is a bug.

I've submitted PR https://github.com/envenomator/agon-ez80asm/pull/98 to change this, if it is supposed to return 0 on a successfully printing the version information.

Thanks!

**Comment by envenomator** [comment]
I agree; using the -v option shouldn't continue actions. Thanks!

**Comment by envenomator** [comment]
Released as v1.11
